app/backend/src/couchers/servicers/requests.py

```diff
@@ -105,6 +106,8 @@ def CreateHostRequest(self, request, context):
             host_request.from_last_seen_message_id = message.id
             session.add(host_request)
 
+            send_host_request_email(host_request)
```

Comment:
You probably need to add a `session.commit()` above this line, otherwise the `host_request` doesn't have all necessary data.